### PR TITLE
Update celo-recovery.md

### PR DESCRIPTION
**Pull request by dominiquejane**
Completes instructions for recovery via MyEtherWallet and adds instructions for another option - MyCrypto.
Edits instructions so they apply to any ERC-20 token rather than just Ethereum.

### CODE REVIEW

File: packages/docs/celo-holder-guide/celo-recovery.md [hunk 1]
```diff
@@ -1,28 +1,51 @@
-# Recovering Ethereum from a Celo Address
+# Recovering ERC-20 tokens sent to a Celo Address
```
Comment (medhakothari) [inline packages/docs/celo-holder-guide/celo-recovery.md]: should it be 
```suggestion
# Recovering ETH or ERC-20 tokens sent to a Celo Address
```
?

File: packages/docs/celo-holder-guide/celo-recovery.md [hunk 2]
```diff
@@ -1,28 +1,51 @@
...
 
-In this guide, steps are outlined to help you recover your [Ethereum](https://en.wikipedia.org/wiki/Ethereum#Addresses) if you accidentally transferred them to a [CELO (previously Celo Gold)](../overview.md#background-and-key-concepts).
+In this guide, steps are outlined to help you recover your ERC-20 token if you accidentally transferred them to a [CELO (previously Celo Gold)](../overview.md#background-and-key-concepts) address.
```
Comment (medhakothari) [inline packages/docs/celo-holder-guide/celo-recovery.md]: see above

**Review by medhakothari — APPROVED** [review]
LGTM! left 2 small comments

File: packages/docs/celo-holder-guide/celo-recovery.md (hunk 1 above)
Comment (dominiquejane) [reply packages/docs/celo-holder-guide/celo-recovery.md]: Sure, that  probably makes it clearer!

**Review by barbaraliau — APPROVED** [review]
LGTM

**Pull request merged**